### PR TITLE
Rename mc/te/srl to full names in apis.

### PR DESCRIPTION
**Pull request by schmmd**

### CODE REVIEW

**Comment by schmmd** [comment]
Happy birthday Joel!

File: allennlp/service/predictors/predictor.py [hunk 1]
```diff
@@ -72,9 +72,9 @@ def default_params() -> Dict[str, Params]:
         decomposable_attention_config = Params(replace_none(config))
 
     return {
-            'mc': bidaf_config,
-            'srl': srl_config,
-            'te': decomposable_attention_config
+            'machine-comprehension': bidaf_config,
+            'semantic-role-labelling': srl_config,
```
Comment (joelgrus) [inline allennlp/service/predictors/predictor.py]: can we do `semantic-role-labeling`?

https://en.wikipedia.org/wiki/Semantic_role_labeling

File: allennlp/service/predictors/semantic_role_labeler.py [hunk 2]
```diff
@@ -8,7 +8,7 @@
 
 import spacy
 
-@Predictor.register("srl")
+@Predictor.register("semantic-role-labelling")
```
Comment (joelgrus) [inline allennlp/service/predictors/semantic_role_labeler.py]: same comment

File: allennlp/service/static/index.html [hunk 3]
```diff
@@ -790,7 +790,7 @@ <h1 className="header__content__logo">
     });
 
     var payload = { sentence: inputs.sentenceValue };
-    fetch('/predict/srl', {
+    fetch('/predict/semantic-role-labelling', {
```
Comment (joelgrus) [inline allennlp/service/static/index.html]: same comment

File: tests/commands/bulk_test.py [hunk 4]
```diff
@@ -40,7 +40,7 @@ def test_works_with_known_model(self):
             f.write("""{"sentence": "this is a less great sentence"}\n""")
 
         args = ["bulk",     # command
-                "srl", # model_name
+                "semantic-role-labelling", # model_name
```
Comment (joelgrus) [inline tests/commands/bulk_test.py]: same comment

File: tests/service/server_sanic_test.py [hunk 5]
```diff
@@ -16,32 +16,32 @@ class TestApp(AllenNlpTestCase):
...
                                        json={"premise": "the super bowl was played in seattle",
                                              "hypothesis": "the super bowl was played in ohio"})
         assert response.status == 200
         results = json.loads(response.text)
         assert "label_probs" in results
 
-    def test_srl(self):
-        _, response = self.client.post("/predict/srl",
+    def test_semantic_role_labelling(self):
+        _, response = self.client.post("/predict/semantic-role-labelling",
```
Comment (joelgrus) [inline tests/service/server_sanic_test.py]: same comment

**Comment by schmmd** [comment]
Joel, you crack me up...

**Pull request merged**